controllers/comment.controller.js

```diff
@@ -0,0 +1,24 @@
+const db = require('../models');
+
+module.exports = {
+  listByPost: async (req, res) => {
+    // const variables
+    const { id: post_id } = req.params;
+    try {
+      const comments = await db.Comment.findAll({
+        where: { post_id },
```

Comment:
post_id no seria id?

Reply:
Recibe ese nombre porq es así como se llama la clave foránea en la tabla de comentarios.

Reply:
Bien Ema, tenes razon, me confundi!